From e5ccf096569244af25820f7d8d94893fff00ce8c Mon Sep 17 00:00:00 2001
From: Henri Yandell <bayard@apache.org>
Date: Fri, 18 Dec 2009 07:21:10 +0000
Subject: [PATCH] Switching from getTime() to getTimeInMillis() per Sebb's
 report that it still fixes LANG-538 and is going to be lighter weight

git-svn-id: https://svn.apache.org/repos/asf/commons/proper/lang/trunk@892161 13f79535-47bb-0310-9956-ffa450edef68
---
 src/java/org/apache/commons/lang3/time/FastDateFormat.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/java/org/apache/commons/lang3/time/FastDateFormat.java b/src/java/org/apache/commons/lang3/time/FastDateFormat.java
index d60af961093..e30da2a5685 100644
--- a/src/java/org/apache/commons/lang3/time/FastDateFormat.java
+++ b/src/java/org/apache/commons/lang3/time/FastDateFormat.java
@@ -869,7 +869,7 @@ public StringBuffer format(Date date, StringBuffer buf) {
      */
     public StringBuffer format(Calendar calendar, StringBuffer buf) {
         if (mTimeZoneForced) {
-            calendar.getTime(); /// LANG-538
+            calendar.getTimeInMillis(); /// LANG-538
             calendar = (Calendar) calendar.clone();
             calendar.setTimeZone(mTimeZone);
         }
